Caltech-IPAC/firefly
# IRSA-2129: fixed bug, now accurate down to .02 arcsec

fixed bugs in IRSA-2129.

_to test:_

https://irsawebdev9.ipac.caltech.edu/IRSA-2129-hips-zoom/firefly/

See ticket: https://jira.ipac.caltech.edu/browse/IRSA-2129
